SimPEG/EM/FDEM/SrcFDEM.py

```diff
@@ -1,65 +1,31 @@
-from SimPEG import Survey, Problem, Utils, np, sp
+import properties
 from scipy.constants import mu_0
-from SimPEG.EM.Utils import (omega, MagneticDipoleFields,
-                             MagneticDipoleVectorPotential,
-                             MagneticLoopVectorPotential,
-                             orientationDict)
-from SimPEG.Utils import Zero
 import warnings
 
+from SimPEG.Utils import Zero
+from SimPEG import Survey, Problem, Utils, np, sp
+from SimPEG.EM.Utils import (
+    omega, MagneticDipoleFields, MagneticDipoleVectorPotential,
```

Comment:
in future, can we adopt a pattern of importing things as a package rather than components from the package?

`import SimPEG.EM.Utils as emutils`

This will help us break the package apart in the future.